pkg/auth/authentication/auth_test.go

```diff
@@ -819,3 +822,49 @@ func (suite *AuthSuite) TestAuthorizeUnknownUserAdminLogsIn() {
 	suite.Equal(adminUser.ID, session.AdminUserID)
 	suite.Equal(uuid.Nil, session.OfficeUserID)
 }
+
+type MockAPIContext struct{}
+
+func (m MockAPIContext) RouteInfo(r *http.Request) (*middleware.MatchedRoute, *http.Request, bool) {
+	matchedRouteMiddleware := middleware.MatchedRoute{}
+	matchedRouteMiddleware.Operation = &spec.Operation{}
+	matchedRouteMiddleware.Operation.VendorExtensible = spec.VendorExtensible{}
+	matchedRouteMiddleware.Operation.VendorExtensible.Extensions = spec.Extensions{}
+	matchedRouteMiddleware.Operation.VendorExtensible.Extensions["x-swagger-roles"] = []interface{}{"office", "contractingOfficer", "customer"}
+	return &matchedRouteMiddleware, nil, false
+}
+
+func (suite *AuthSuite) TestRequireRoleAuthMiddleware() {
```

Comment:
should we update name of function to capture that this will fail?
```suggestion
func (suite *AuthSuite) TestRequireWrongRoleAuthMiddleware() {
```

Reply:
(or something like this)

Reply:
??? It should be passing. The roles match and everything.